docs/release-notes.rst

```diff
@@ -1,3 +1,289 @@
+
+Workaround: Remove the duplicate IPs from the endpoint list on APIC and retry a manual sync of Endpoints from L4-L7 App Services --> Application Inventory --> Sync EPs icon. 
+
+Note: Similar issues might be seen with other erroneous configurations such as unsupported IPv4 formats like 1.2.3.4/24 instead of 1.2.3.4
+
+AS3 Defect: https://github.com/F5Networks/f5-appsvcs-extension/issues/287
+
+------
+
+**Pool members deleted or added directly to BIG-IP don't get updated automatically after clicking "Sync EPs"**
+
+1. If BIG-IP pool members are automatically added by the **Dynamic endpoint discovery** feature, but then a few endpoints are deleted directly from the BIG-IP (i.e. out of band); these endpoints do not get created again when clicking **L4-L7 App Services --> Application Inventory --> Sync EPs** for that application. 
+
+2. Similarly, when a few pool members are added directly to the BIG-IP (i.e. out of band), these extra members are not deleted after clicking **L4-L7 App Services --> Application Inventory --> Sync EPs**
+
+**Workaround:** Manually add/delete the pool members from BIG-IP. 
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/293
+
+------
+
+**Pool members are not synced on AS3 service discovery REST API endpoint for HA devices**
+
+AS3 Service Discovery REST API endpoint on both HA devices should display the same pool member list for the specified pool path (For. ex. https://BIG-IP/mgmt/shared/service-discovery/task/~Partition~Application~Pool). But AS3 service discovery fails to perform this sync between the HA devices.
+
+**Workaround:** None.
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/385
+
+------
+
+**FAST input fields do not display any validation errors until the field is touched**
+
+Due to Angular Forms behaviour, for FAST input fields that have any validations, display a validation error message only when that field is clicked/touched explicitly. There is no indication given to the user for the validations that are applied to these fields. 
+
+1. Blue Pool Name, Green Pool Name - Blue Green Template
+2. Host name - DNS Template
+3. Password - LDAP Template
+4. Application Domain Name - Microsoft Sharepoint Template
+5. Domain Name - SMTP Template
+
+**Workaround**: FAST input fields can be clicked for the validation error message to be displayed.
+
+------
+
+**Bluegreen Template FAST form displays some extra fields within Service Discovery Type which are not displayed on BIG-IP FAST Form**
+
+As in the latest FAST plugin v1.14 a different format for defining dependencies, the following FAST input fields (API Key, Application ID, Directory ID, FQDN, Region, Resource Group, Resource ID, Resource Type, Subscription ID, tagKey, tagValue, URI) were not evaluated correctly on the UI. Due to this these are displayed on the FASC FAST form whereas the BIG-IP UI does not display them by default. 
```

Comment:
The latest FAST plugin (v1.14) introduced a different format for defining dependencies, so the following FAST input fields are not evaluated correctly in the UI: API Key, Application ID, Directory ID, FQDN, Region, Resource Group, Resource ID, Resource Type, Subscription ID, tagKey, tagValue, URI.  Because of this, these fields are displayed...